src/BinDeps.jl

```diff
@@ -1,37 +1,32 @@
 module BinDeps
     importall Base
 
+    using Compat
+
     export @make_run, @build_steps, find_library, download_cmd, unpack_cmd,
             Choice, Choices, CCompile, FileDownloader, FileRule,
             ChangeDirectory, FileDownloader, FileUnpacker, prepare_src,
             autotools_install, CreateDirectory, MakeTargets, SystemLibInstall
 
-    if VERSION >= v"0.4.0-dev+3844"
-        import Base.Libdl: dlext, dlpath, RTLD_LAZY, DL_LOAD_PATH
-    else
-        const dlext = isdefined(Base.Sys, :shlib_ext) ? Base.Sys.shlib_ext : Base.Sys.dlext # Julia 0.2/0.3 compatibility
```

Comment:
or maybe move this line of code to [Compat.jl](https://github.com/JuliaLang/Compat.jl/blob/54ab44412261b7e1de9d0b47242790ac27b13d9e/src/Compat.jl#L280), since 0.2's `Base.Sys` did not have a `dlext`